cmd/watch.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/pulumi/pulumi/pkg/backend"
+	"github.com/pulumi/pulumi/pkg/backend/display"
+	"github.com/pulumi/pulumi/pkg/engine"
+	"github.com/pulumi/pulumi/pkg/util/cmdutil"
+	"github.com/pulumi/pulumi/pkg/util/result"
+)
+
+// intentionally disabling here for cleaner err declaration/assignment.
+// nolint: vetshadow
+func newWatchCmd() *cobra.Command {
+	var debug bool
+	var expectNop bool
+	var message string
+	var stack string
+	var configArray []string
+	var path bool
```

Comment:
Nit: I ended up changing this variable to `configPath` elsewhere to make it a little more clear that this is related to `configArray`. Would be good to do so here as well.

```suggestion
	var configPath bool
```

Reply:
Fixed.